packages/boxed-expression-component/src/i18n/locales/en.ts

```diff
@@ -112,4 +112,70 @@ export const en: BoxedExpressionEditorI18n = {
   selectExpression: "Select expression",
   selectFunctionKind: "Select Function Kind",
   selectLogicType: "Select logic type",
+  contextExpression: {
+    variable: "variable",
+    expression: "expression",
+  },
+  yourAnnotationsHere: "// Your annotations here",
+  hitpolicy: "Hit policy",
+  aggregatorFunction: "Aggregator function",
+  logicTypeHelp: {
+    literExpression:
```

Comment:
```suggestion
    literal:
```

following the next properties which doesn't use the `expression` on their names